### PR TITLE
Adding the "fauxpose" plugin (again)

### PR DESCRIPTION
**Pull request by wsherman64**
Updated plugin code and comments based on past pull request comments.  Decided to resubmit as a new PR.

### CODE REVIEW

**Comment by charmoniumQ** [comment]
lgtm

File: configs/faux.yaml [hunk 1]
```diff
@@ -0,0 +1,40 @@
+plugin_groups:
+  #- !include "rt_slam_plugins.yaml"
```
Comment (Jebbly) [inline configs/faux.yaml:2]: I think it's fine to remove the commented plugins for clarity.

File: fauxpose/common
Comment (Jebbly) [inline fauxpose/common:1]: Looks like this is a hardcoded path, which should be replaced with `../common`.

**Review by Jebbly — CHANGES_REQUESTED** [review]

File: fauxpose/README_fauxpose.md [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+values set to rotate in a circle.  The period and amplitude of the movement
+are have default values (0.5, and 2.0 respectively) that can be changed
+through environment variables.
+
+    Topic details:
+    -   *Publishes* `pose_position` on `fast_pose` topic.
+
+## Usage
+The "FauxPose" plugin must be included in the YAML configuration file
+prior to any rendering plugin (such as "debugview" or "gldemo").  Also
```
Comment (rsingh76) [inline fauxpose/README_fauxpose.md:19]: Will this work with monado (OXR) based applications?
If yes, could you change the config files to give that functionality as well?

**Review by jianxiapyh — APPROVED** [review]

File: docs/README_fauxpose.md [hunk 3]
```diff
@@ -0,0 +1,38 @@
...
+
+## Summary
+The `FauxPose` ILLIXR plugin generates "fast_pose" data from a simple
+mathematical algorithm (circular movement).  The intent is for use when
+debugging other plugins and the developer wants a known pose trajectory
+without having to configure actual tracking.
+
+The movement is hard-coded to be on the Y=center.y plane, with the X and Z
+values set to rotate in a circle.  The period and amplitude of the movement
+are have default values (0.5, and 2.0 respectively) that can be changed
```
[the remark this answers is not in the document]
Comment (jianxiapyh) [reply docs/README_fauxpose.md:11]: I tested on my machine and works

**Review by Jebbly — APPROVED** [review]

File: fauxpose/README_fauxpose.md (hunk 2 above)
Comment (Jebbly) [reply fauxpose/README_fauxpose.md:19]: Seems like `configs/faux.yaml` is more of a sample to use, and documentation on how to use it should be sufficient.

File: configs/faux.yaml [hunk 4]
```diff
@@ -0,0 +1,34 @@
+plugin_groups:
+  - plugin_group:
+      - path: fauxpose/
+      - path: gldemo/
+      - path: debugview/
+  - !include "core_plugins.yaml"
+
+data:
+  subpath: mav0
```
[the remark this answers is not in the document]
Comment (Jebbly) [reply configs/faux.yaml:9]: Yeah, it doesn't make sense to use in this context. Could possible be left for the CMake PR.

File: docs/README_fauxpose.md (hunk 3 above)
[the remark this answers is not in the document]
Comment (Jebbly) [reply docs/README_fauxpose.md:11]: For the sake of getting this merged, let's fix this in the update documentation PR.

**Review by Boyuan-Tian — APPROVED** [review]

**Review by rsingh76 — APPROVED** [review]

**Pull request merged**